File: android/CustomControls/src/main/java/com/everysight/sdk/samples/customcontrols/SimpleControl.kt
package com.everysight.sdk.samples.customcontrols

import UIKit.app.data.EvsColor
import UIKit.widgets.Frame
import UIKit.widgets.Rect

class SimpleControl:Frame() {

    override fun onCreate() {
        super.onCreate()

        add(Rect()
            .setWidth(50f)
            .setHeight(50f)
            .setForegroundColor(EvsColor.Blue.rgba)
            .setBackgroundColor(EvsColor.Blue.rgba))

        add(Rect().setWidth(50f)
            .setHeight(50f)
            .setForegroundColor(EvsColor.Green.rgba)
            .setBackgroundColor(EvsColor.Green.rgba)
            .rotate(45f))
    }

}

File: ios/Animations/Animations/Glasses/SimpleControl.swift
//Copyright (c) 2020 Everysight LTD

import Foundation
import NativeEvsKit

class SimpleControl:Frame {
    
    override func onCreate() {
        super.onCreate()
        let rect = Rect()
        rect.setWidth(width: 50.0)
            .setHeight(height: 50.0)
            .setForegroundColor(color: EvsColor.blue.rgba)
            .setBackgroundColor(color: EvsColor.blue.rgba)
        add(uiElement: rect)
        
        let rect2 = Rect()
        rect2.setWidth(width: 50.0)
            .setHeight(height: 50.0)
            .setForegroundColor(color: EvsColor.green.rgba)
            .setBackgroundColor(color: EvsColor.green.rgba)
            .rotate(angleDegrees: 45.0)
        add(uiElement: rect2)
    }
    
}
